server/src/test/java/org/elasticsearch/client/documentation/IndicesDocumentationIT.java

```diff
@@ -89,6 +93,11 @@ public void testPutMappingDocumentation() throws Exception {
                 "}", XContentType.JSON)
         .get();
         // end::putMapping-request-source
+        getMappingsResponse = client.admin().indices().prepareGetMappings("twitter").get();
+        assertEquals(1, getMappingsResponse.getMappings().size());
+        indexMapping = getMappingsResponse.getMappings().get("twitter");
+        assertThat(indexMapping.get("user"), instanceOf(MappingMetaData.class));
```

Comment:
do we really need this assertion, given the one immediately following? this feels like it could be an implementation detail.